sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BatchExecUtils.java

```diff
@@ -52,6 +52,15 @@ public static String getActivityId(Map<String, String> responseHeaders) {
         return null;
     }
 
+    public static int getResponseLength(Map<String, String> responseHeaders) {
+        int responseLength = 0;
+        if (responseHeaders != null) {
+            responseLength = Integer.parseInt(responseHeaders.get(HttpConstants.HttpHeaders.CONTENT_LENGTH));
```

Comment:
Put it inside a try catch just to be safe. If somehow it is not present(may be in case of non-success status), we should not throw exception to the user. See how Request Charge is parsed.